libs/shared/form-fields/src/lib/CompanySearchController/CompanySearchController.tsx

```diff
@@ -3,21 +3,10 @@ import { Controller, useFormContext } from 'react-hook-form'
 import { AsyncSearch, Box, AlertMessage } from '@island.is/island-ui/core'
```

Comment:
This component is probably the root issue. It should not be in this library, since it contains side effects that around bound to all applications.